code/modules/mob/mob.dm

```diff
@@ -9,7 +9,10 @@
 		for(var/atom/movable/AM in client.screen)
 			qdel(AM)
 		client.screen = list()
-	ghostize()
+
+	//Only living mobs can become ghosts
+	if (istype(src, /mob/living))
+		ghostize()
```

Comment:
We shouldn't remove it here. 
If something delete ghost it (ghost player) will stuck in nullspace/GC.

Reply:
That isn't "Ghost me proc" that is safe-call to detach client/player from mob.

Reply:
doing